decentraland/screens/MyAssetsScreen.js

```diff
@@ -46,10 +46,12 @@ export class MyAssetsScreen extends React.Component {
   _getAssetsAndActionsFromBlockchain = async address => {
     const assetsFromBlockchain = await this._getAssetsFromBlockchain(address);
 
-    const actionsFromBlockchain = assetsFromBlockchain.map(asset => {
+    let actionsFromBlockchain = {};
+
+    assetsFromBlockchain.forEach(asset => {
       const { actionId, id: assetId } = asset;
-      const status = SUCCESSFUL;
-      return { actionId, assetId, status };
+      const userAction = { [actionId]: { assetId, status: SUCCESSFUL } };
```

Comment:
Why is `actionId` in `[]` brackets here?

Reply:
To use the `actionId` value as key:

`{ [actionId]: { assetId, status: SUCCESSFUL } };` => `{ '0x123...789': { assetId, status: SUCCESSFUL } };`
`{ actionId: { assetId, status: SUCCESSFUL } };` => `{ actionId: { assetId, status: SUCCESSFUL } };`

Reply:
Ah okay, I was guessing that might be it. Thanks for the explanation!

Resolving this.